### PR TITLE
Added tamuhack routing for old tamuhack

### PR DESCRIPTION
**Pull request by jonathan-innis**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -22,6 +24,7 @@ app.engine("html", ejs.renderFile);
 app.get("/", hubRoute);
 app.get("/hh", howdyHackRoute);
 app.get("/workshops", workshopRoute);
+app.get("/th2019", th2019Route);
```
Comment (SaltyQuetzals) [inline src/app.ts]: Please make an [express router](https://expressjs.com/en/guide/routing.html) that we can use for additional future/past versions.

Something along the lines of: `https://tamuhack.com/th/<YEAR>`, and something similar for HowdyHack.

Thoughts @gracekfan ?

Comment (SaltyQuetzals) [inline src/app.ts]: Please make an [express router](https://expressjs.com/en/guide/routing.html) that we can use for additional future/past versions.

Something along the lines of: `https://tamuhack.com/th/<YEAR>`, and something similar for HowdyHack.

Thoughts @gracekfan ?

Comment (gracekfan) [reply src/app.ts]: Might be confusing when we do routing for HowdyHack, but would both /hh and /hh/2019 redirect to current? And then /hh/2018, etc?

Comment (gracekfan) [reply src/app.ts]: Also were you able to ask Jay about the TH sites before 2019? @jonathan-innis

Comment (jonathan-innis) [reply src/app.ts]: Going to contact him today....wanted to get this done first

Comment (jonathan-innis) [reply src/app.ts]: Will change how routing works later this evening

Comment (SaltyQuetzals) [reply src/app.ts]: We'd be removing the`/hh` route for this

Comment (gracekfan) [reply src/app.ts]: Wait I don’t agree with this routing method then. I think the current iteration should be different from past ones/easy to get to

Comment (gracekfan) [reply src/app.ts]: Lol okay

Comment (SaltyQuetzals) [reply src/app.ts]: Okay, how about `/th/` and `/hh/` for current, and then the others have years?

Comment (jonathan-innis) [reply src/app.ts]: yeah we could also just do a redirect on the route for /th/2019 if we want to /th

Comment (jonathan-innis) [reply src/app.ts]: Yeah I'm with Grace ^^^ I think it should be /th and /hh with older version having /th/2019, /th/2018, etc.

**Pull request closed without merging**